min support, but much much shorter (#83)
quicwg/version-negotiation: draft-ietf-quic-version-negotiation.md

remain stable for the entire handshake (as QUIC version 1 does), then this
applies to the entire handshake, including the first flight.

Note also that the client can disable compatible version negotiation by

> omitting the Version Information Transport Parameter, or not reporting other
> versions in it.

Reviewer: Omitting the transport parameter removes the version from the TLS transcript which weakens security. I would instead:
```suggestion
only including the Chosen Version in the Other Versions field of the Version
Information Transport Parameter.
```
